README.md

```diff
@@ -9,7 +9,16 @@ Der Bezug ist über ein Abonnement möglich. Kostenlose Demoausgaben bieten die
 Weitere Infos unter [http://www.taz.de/android](http://www.taz.de/android)
 
 
-[![Android app on Google Play](https://developer.android.com/images/brand/de_app_rgb_wo_60.png)](https://play.google.com/store/apps/details?id=de.thecode.android.tazreader)
+<a href="https://play.google.com/store/apps/details?id=de.thecode.android.tazreader">
+    <img alt="Get it on Google Play"
+        height="80"
+        src="https://user-images.githubusercontent.com/15004217/36810046-fa306856-1cc9-11e8-808e-6eb8a81783c7.png" />
```

Comment:
Why get the images from githubusercontent?

Reply:
I had two ideas on my mind:
- Hosting: When a website goes down, the badges are not shown. F-Droid had some troubles the last days (see [here e.g.](https://forum.f-droid.org/t/why-is-fdroid-down-all-the-time/5028)) and I missed the F-Droid badge on some repos here on GitHub. 
_(The [Google](https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png) website is always online (0% downtime and always extremely fast) though.)_
- Privacy: Linking the official badges from F-Droid and especially [Google](https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png) would give information to F-Droid and Google about visitors of your GitHub repo. I think that F-Droid is not tracking that much and I could live with that personally but I prefer not to send my information to Google.

Reply:
Sounds reasonable. I'm not used to githubusercontent though.